src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AzureFunction/AzureFunctionIsolatedInvokeFunctionAsyncWrapper.cs

```diff
@@ -58,8 +58,8 @@ public AfterWrappedMethodDelegate BeforeWrappedMethod(InstrumentedMethodCall ins
             throw new Exception("FunctionDetails are missing some require information.");
         }
 
-        agent.RecordSupportabilityMetric($"DotNet/AzureFunction/Worker/Isolated");
-        agent.RecordSupportabilityMetric($"DotNet/AzureFunction/Trigger/{functionDetails.TriggerTypeName ?? "unknown"}");
+        agent.RecordSupportabilityMetric($"Dotnet/AzureFunction/Worker/Isolated");
```

Comment:
I'm not sure if there's any performance penalty for having these be interpolated strings (starting with `$`) but if there is we should change them to literals.

Reply:
Those are strays from a previous iteration where there was going to be some actual string interpolation. I'll clean them up.